File: example/androidlib/java/1-hello-world/app/src/androidTest/java/com/helloworld/app/SampleLogicInstrumentedTest.java
package com.helloworld.logictest;

import static org.junit.Assert.*;

import androidx.test.ext.junit.runners.AndroidJUnit4;
import com.helloworld.SampleLogic;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(AndroidJUnit4.class)
public class SampleLogicInstrumentedTest {
  @Test
  public void testTextSize() {
    float textSize = SampleLogic.textSize();
    assertEquals(32.0f, textSize, 0.0001f);
  }
}

File: example/androidlib/kotlin/1-hello-kotlin-declarative/app/src/androidTest/java/com/helloworld/app/SampleLogicInstrumentedTest.kt
package com.helloworld.logictest

import androidx.test.ext.junit.runners.AndroidJUnit4
import com.helloworld.SampleLogic
import org.junit.Assert.*
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class SampleLogicInstrumentedTest {
    @Test
    fun text_size_is_correct() {
        assertEquals(32f, SampleLogic.textSize(), 0.0001f)
    }
}
